src/GitLink/Program.cs

```diff
@@ -24,51 +27,51 @@ private static int Main(string[] args)
             var consoleLogListener = new OutputLogListener();
             LogManager.AddListener(consoleLogListener);
 
-            try
+            Uri remoteGitUrl = null;
+            string pdbPath = null;
+            bool skipVerify = false;
+            bool downloadWithPowershell = false;
+            var arguments = ArgumentSyntax.Parse(args, syntax =>
             {
-                HelpWriter.WriteAppHeader(s => Log.Write(LogEvent.Info, s));
+                syntax.DefineOption("u|url", ref remoteGitUrl, s => new Uri(s, UriKind.Absolute), "Url to remote git repository.");
```

Comment:
The reason we have branch and commitid is for dynamic repositories. When using this tool on a build agent that doesn't have access to the .git directory, we create dynamic repositories on disk using this information. However, if we have the commitid, that should be sufficient for creating the srcsrv and update the pdb files. I think we used the dynamic repositories to automatically select the latest commit id if it wasn't available. However, I think we can drop this because the right approach is the always specific the commit id a build is representing.